### ISSUE TITLE
Instrument, InstrumentDesign, InstrumentInitiative

### ISSUE THREAD

**Issue by mrpatrickwest** [tetherless-world/dco-ontology#63]
@olyerickson , @zednis , @xgmachina , @zhonghao1014 , @am-e 

Discussion about instrument schema

Current cmap: http://cmapspublic3.ihmc.us/rid=1PSRC828N-1X79HZJ-1G13/dco-instrument-model-new.cmap

Currently I have in there Instrument, which we agree is the physical device, which has a type InstrumentType. But I don't like that term InstrumentType. It's more of a design and should be called InstrumentDesign.

Then there's what Engagement and Secretariat call InstrumentInitiative which relates the other two concepts, so maybe

InstrumentInitiative hasDesign InstrumentDesign ; buildsInstrument Instrument .

or something like that.

**Comment by mrpatrickwest** [comment]
inverse properties

Instrument builtForInitiative InstrumentInitiative .
InstrumentDesigned designedForInitiative InstrumentInitiative .

**Comment by zednis** [comment]
I think we should discuss the purpose for what we currently call InstrumentType.

The purpose for this is to be able to ~~classify~~ categorize (perhaps not formally via RDFS reasoning) Instruments, correct?

edit: perhaps a simple use case is in order.

"list all Spectrometers used by the deep carbon community"

A theoretical instrument browser would have a facet which would have an entry of value "Spectrometer (n)" and when selected the browser would show a list of Instruments that are spectrometers.

What is this facet called?  What property is used in the model to populate this facet?

I don't think this is InstrumentIniative or InstrumentModel.  I would say it is a general "type" for the instrument which is why it was originally called InstrumentType and the facet would probably just be called "Type".

Would it make sense for this facet to be "Design"?  I don't know, that seems less natural to me.

**Comment by mrpatrickwest** [comment]
No, not really. A Sloan funded project could just be funded to design an instrument. For example, the Push 50 instrument was designed as part of funded projec.. A separate project funded by Sloan built the two physical devices. So we needed a representation of Push 50, the designed instrument that we were originally calling InstrumentType, and the two physical devices created, what we are calling Instrument.

**Comment by zednis** [comment]
I think design, initiative, model, etc are all useful properties that we should explore, but I think they go beyond the original intent of InstrumentType which was to provide a simple way to categorize Instruments.

TL;DR I think the scope of what was originally InstrumentType has been expanded into something else.

Why not have an InstrumentInitiative and InstrumentDesign separate from InstrumentType?

Sloan funding the design of a specific Spectrometer does not mean the general category spectrometer was funded by Sloan.

**Comment by zednis** [comment]
I think the original intent of InstrumentType and InstrumentModel (funded by Sloan and designed via an InstrumentInitiative) are different things.

**Comment by zednis** [comment]
I propose that InstrumentType and InstrumentModel (or InstrumentDesign) are different things.

**Comment by zednis** [comment]
In Patrick's example above, PUSH50 is an InstrumentModel/InstrumentDesign, funded by sloan and developed via an InstrumentInitiative.

The InstrumentType would be broader, perhaps "Underwater Sampler".

**Comment by mrpatrickwest** [comment]
Okay ... the scope of this exercise is this: Currently engagement and secretariate maintain a static page in drupal that lists what they call Instrument Initiatives: https://deepcarbon.net/page/instrumentation-initiatives

The task is to dynamically generate this page, and for each instrument on that page link to a summary page. All dynamically generated from the TS

We agreed that Instrument represents the physical device, which I think it still does. These are the built devices. The summary page  would have links to the individual physical devices, each of which would have a point of contact so that people could reserve/use the physical device. This is the thing with the serial number.

So what to call the things on that instrument initiative page. At first we thought it would be the thing with the model infromation. IN our example, Push 50. And we called this InstrumentType ... 

Today's discussion ...

but it's not an instrument type it's the design of the instrument. There is no physical device here, just a design. So I'd like to call that the InstrumentDesign.

And  together, the design and the physical devices, represent the instrument initiative. And it's the instrument initiatve that gets listed on that instrument initiative page. The summary page will include information about the design of the instrument, and the construction of phyusical devices.

**Comment by zednis** [comment]
I agree that what you have described is not the same as the original intent of InstrumentType and I do not think what we currently call InstrumentType should be reinterpreted to this scenario.  So :+1: to a new class InstrumentDesign, which is not the same as InstrumentType.

Do you think it would make sense for the InstrumentDesign to reference a InstrumentType?  (I do, but I would make it optional)

Looking at the current Deepcarbon instrument initiatives page I see 5 initiatives that built designs that from the design name appear to be Spectrometers.  That would be 5 different InstrumentDesigns that I think would have instrumentType Spectrometer (or a subtype of it if that was deeemed necessary).

**Comment by mrpatrickwest** [comment]
:+1:

**Comment by mrpatrickwest** [comment]
@olyerickson @xgmachina 

What do you both think? Just read the last two posts and I think you'll agree.

**Comment by am-e** [comment]
I agree with the conclusion of Instrument, InstrumentDesign, and
InstrumentInitiative, and I think it fits with the original scope.

I was previously under the impression that our intent was mainly to
categorize instruments, for which InstrumentType was meaningful. So now,
where would the current instruments fit in?

**Comment by zednis** [comment]
I think an Instrument can have both InstrumentType and InstrumentDesign; both are meaningful for categorization.

If I was to use a car analogy I would say that "Truck" was a type and "F-350" was a design.

**Comment by mrpatrickwest** [comment]
New CMAP: http://cmapspublic3.ihmc.us/rid=1PSRC828N-1X79HZJ-1G13/dco-instrument-model-new.cmap

**Comment by mrpatrickwest** [comment]
Change Proposal document updated: https://docs.google.com/document/d/1vqkxQ48BLwkS60Vcdj_H1MVvicaTM2AtGjyc_gNdL2o/edit
